blocks/library/text-columns/test/__snapshots__/index.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`core/text-columns block edit matches snapshot 1`] = `
 <div
-  class="wp-block-text-columns alignundefined columns-2"
+  class="wp-block-text-columns columns-2"
```

Comment:
`alignundefined` doesn't make too much sense, that's why used `classnames` to fix this.